<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>920</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IoT Server — Edges For Training</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <property name="styleSheet">
        <string notr="true">#tab_2 {
    border-image: url(:/new/prefix1/Imagesandicons/Background.jpg) 0 0 0 0 stretch stretch;
}</string>
       </property>
       <attribute name="title">
        <string>Real Time Monitor</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true">#tab {
    border-image: url(:/new/prefix1/Imagesandicons/Background.jpg) 0 0 0 0 stretch stretch;
}</string>
       </property>
       <attribute name="title">
        <string>Historical Analysis</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="styleSheet">
        <string notr="true">#tab_3 {
    border-image: url(:/new/prefix1/Imagesandicons/Background.jpg) 0 0 0 0 stretch stretch;
}
QLabel {
    color: white;
    font-size: 14px;
}
QSlider::groove:horizontal {
    height: 8px;
    background: #2d2d44;
    border-radius: 4px;
}
QSlider::handle:horizontal {
    background: #2ecc71;
    width: 22px;
    height: 22px;
    margin: -7px 0;
    border-radius: 11px;
}
QSlider::sub-page:horizontal {
    background: #2ecc71;
    border-radius: 4px;
}
QLCDNumber {
    background: #16213e;
    color: #2ecc71;
    border: 2px solid #0f3460;
    border-radius: 6px;
    font-size: 32px;
}</string>
       </property>
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="13" column="2">
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>60</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>200</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>TCP</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>220</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="12" column="2">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel { font-size:15px; font-weight:bold; padding:8px; }</string>
            </property>
            <property name="text">
             <string>Current Threshold (°C):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">QLabel { font-size:15px; font-weight:bold; padding:8px; }</string>
            </property>
            <property name="text">
             <string>Slide to configure the temperature threshold:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="10" column="2">
           <widget class="QSlider" name="horizontalSlider">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TickPosition::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>220</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QLabel { font-size:15px; font-weight:bold; padding:8px; }</string>
            </property>
            <property name="text">
             <string>Select Connection Type: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="14" column="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>200</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>UDP</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="2" rowspan="2">
           <widget class="QPushButton" name="connectButton">
            <property name="text">
             <string>Connect</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <property name="styleSheet">
        <string notr="true">#tab_4 {
    border-image: url(:/new/prefix1/Imagesandicons/Background.jpg) 0 0 0 0 stretch stretch;
}
QPushButton {
    background-color: transparent;
    border: none;
}
QPushButton:hover {
    background-color: rgba(255,255,255,30);
    border-radius: 12px;
}</string>
       </property>
       <attribute name="title">
        <string>Quick Access</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="toolTip">
             <string>Open Facebook – Edges For Training</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Photos.qrc">
              <normaloff>:/new/prefix1/Imagesandicons/facebook-app-symbol.png</normaloff>:/new/prefix1/Imagesandicons/facebook-app-symbol.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="toolTip">
             <string>Open LinkedIn – Edges For Training</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Photos.qrc">
              <normaloff>:/new/prefix1/Imagesandicons/linkedin.png</normaloff>:/new/prefix1/Imagesandicons/linkedin.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="toolTip">
             <string>Open Instagram – Edges For Training</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Photos.qrc">
              <normaloff>:/new/prefix1/Imagesandicons/instagram.png</normaloff>:/new/prefix1/Imagesandicons/instagram.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Photos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_2</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
